interfaces/encoding.idl

```diff
@@ -3,6 +3,12 @@
 // (https://github.com/tidoust/reffy-reports)
 // Source: Encoding Standard (https://encoding.spec.whatwg.org/)
 
+interface mixin TextDecoderCommon {
```

Comment:
Hmm, we have an interesting problem here. As the boilerplate says this file is generated, and if these changes are made in the spec they'd eventually propagate here. And yet making the changes in the wpt PR linked to the spec PR is great, makes it easy to understand the source of the changes.

But, if this change is merged before the spec is scraped, an interfaces/ IDL auto-update might try to effectively revert the changes again, and later re-apply them. That would be silly.

Still, please carry on, we should just make tooling handle a situation like this.